deployment/docker/Dockerfile

```diff
@@ -1,53 +1,58 @@
-#--------- Generic stuff all our Dockerfiles should start with so we get caching ------------
-FROM python:2.7.18-buster
-MAINTAINER Admire Nyakudya<admire@kartoza.com>
+# Use the official Python image with version 3.11 as the base image
+FROM python:3.11-buster
 
+# Set the maintainer label
+LABEL maintainer="Admire Nyakudya <admire@kartoza.com>"
 
 # Install system dependencies
 RUN apt-get update && \
-    apt-get install -y libgdal-dev libcurl4-gnutls-dev librtmp-dev libssl-dev  binutils libproj-dev gdal-bin &&  \
-    pip install uwsgi setuptools;pip uninstall -y wheel
+    apt-get install -y libgdal-dev libcurl4-gnutls-dev librtmp-dev libssl-dev binutils libproj-dev gdal-bin && \
+    pip install uwsgi setuptools && pip uninstall -y wheel
 
+# Install Node.js and npm
+RUN curl -sL https://deb.nodesource.com/setup_18.x -o nodesource_setup.sh && \
+    bash nodesource_setup.sh && \
+    apt-get install nodejs -y
 
-RUN curl -sL https://deb.nodesource.com/setup_18.x -o nodesource_setup.sh
-RUN bash nodesource_setup.sh
-RUN apt-get install nodejs -y
-
+# Copy your Django project to /home/web/django_project
 ADD ./django_project /home/web/django_project
+
+# Copy requirements.txt to the root directory
 ADD ./deployment/docker/requirements.txt /REQUIREMENTS.txt
+
+# Install Python dependencies from requirements.txt
 RUN pip install --no-deps -r /REQUIREMENTS.txt
 
-ADD ./deployment/docker/libgeos.py /usr/local/lib/python2.7/site-packages/django/contrib/gis/geos/libgeos.py
-ADD ./deployment/docker/trans_real.py /usr/local/lib/python2.7/site-packages/django/utils/translation/trans_real.py
-#RUN sed -i 's/geos_version().decode()/geos_version().decode().split(\x27 \x27)/g' /usr/local/lib/python2.7/site-packages/django/contrib/gis/geos/libgeos.py
-#USER www-data
-# ADD ./django_project /home/web/django_project
+# Copy necessary files to the appropriate locations
+ADD ./deployment/docker/libgeos.py /usr/local/lib/python3.11/site-packages/django/contrib/gis/geos/libgeos.py
```

Comment:
I think this can be removed as I initially added this to resolve some bug with the specific django and python2

Reply:
Oh i see ,I will update the code